### DISCUSSION TITLE
platform/integrations/zapier

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#1233] [Comments]
# platform/integrations/zapier

Learn how to integrate your Apify Actors with Zapier.

https://docs.apify.com/platform/integrations/zapier

**Comment by lisabecker** [comment]
I've connected Zapier and Apify and have created a Zap task that uses the Apify actor twitterapi/twitter-get-tweetdetail-v2. The Zap task runs successfully and the run is visible on Apify, but I don't see the Apify task output on Zapier. When I test the Zap task, the "Data out" box contains a JSON with 'Output error: Cannot find "OUTPUT" in the key-value store'. I haven't found how to change the output format of an Apify task such that it contains a field "output" with the information from the Apify run, or how to rename the Zap task fields (kind of vice versa). How do I solve this?
